### PR TITLE
Follow up filename replace c_string w/ index lookup

### PR DESCRIPTION
**Pull request by ben-albrecht**
As a followup to #3049, this PR catches some filename arguments that were left as `c_string` instead of `int32_t`s. 

Discovered during exploratory testing of @npadmana's MPI module.

### CODE REVIEW

**Comment by ben-albrecht** [comment]
@ronawho - want to check this?

File: runtime/src/launch/mpirun/launch-mpirun.c [hunk 1]
```diff
@@ -60,13 +60,14 @@ static char* chpl_launch_create_command(int argc, char* argv[],
 
 
 int chpl_launch(int argc, char* argv[], int32_t numLocales) {
+  // if chpl_comm == none, the numLocales = 1
```
Comment (ronawho) [inline runtime/src/launch/mpirun/launch-mpirun.c]: Left this comment in

Comment (ben-albrecht) [inline runtime/src/launch/mpirun/launch-mpirun.c]: crap.

**Comment by ronawho** [comment]
👍

**Pull request merged**